code/messaging/src/main/java/com/adobe/marketing/mobile/messaging/internal/EdgePersonalizationResponseHandler.java

```diff
@@ -119,10 +131,11 @@ class EdgePersonalizationResponseHandler {
                     propositions.addAll(propositionList);
```

Comment:
There is already a global variable named `propositions`. We aren't using it subsequently. Can we remove it or rename this to `propositionsList` to avoid conflict?

Reply:
i've removed the local propositions var as it was not being used. it was a remnant of the old caching code.